ironfish/src/consensus/consensus.ts

```diff
@@ -92,14 +71,18 @@ export class ConsensusParameters {
    */
   V3_DISABLE_MINING_REWARD = Number.MAX_SAFE_INTEGER
 
+  constructor(parameters: ConsensusParameters) {
+    this.parameters = parameters
+  }
+
   isActive(upgrade: number, sequence: number): boolean {
     return sequence >= upgrade
   }
 }
 
-export class TestnetParameters extends ConsensusParameters {
-  constructor() {
-    super()
+export class TestnetConsensus extends Consensus {
+  constructor(parameters: ConsensusParameters) {
+    super(parameters)
     this.V1_DOUBLE_SPEND = 204000
     this.V2_MAX_BLOCK_SIZE = 255000
     this.V3_DISABLE_MINING_REWARD = 279900
```

Comment:
Also doesn't need to be done in this PR, but I wonder if these should be consensus params specified in the network definition file as well.

Reply:
I'm going to figure out how we want to deal with consensus upgrades in a separate PR (part of the Consensus Upgrades project), so I've left these out of the consensus params for now.